workspace/Dockerfile

```diff
@@ -1220,6 +1220,16 @@ RUN if [ ${INSTALL_TERRAFORM} = true ]; then \
     && mv terraform /usr/local/bin \
     && rm terraform_0.10.6_linux_amd64.zip \
 ;fi
+
+###########################################################################
+# Memcached Dependecies:
+###########################################################################
+
+RUN if [ ${PHP_FPM_INSTALL_MEMCACHED} = true ]; then \
```

Comment:
ARG is missing and  this is workspace, not php-fpm.

Reply:
Oh my bad. I have added ARG for workspace.